### PR TITLE
Feature/6 home

### PR DESCRIPTION
**Pull request by dutraneto**
Implemented Home Desktop page.
Please, review code ASAP.

### CODE REVIEW

File: src/templates/components/footer.hbs [hunk 1]
```diff
@@ -1,5 +1 @@
-<footer id="footer" class="site-footer container">
-    <div class="site-author">
```
Comment (marceloglacial) [inline src/templates/components/footer.hbs:2]: Move address info to here

File: src/templates/components/home.hbs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div id="home" class="home">
+    {{> components/sidebar}}
```
Comment (marceloglacial) [inline src/templates/components/home.hbs]: Add header component and move logos to here

File: src/templates/components/sidebar.hbs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<aside id="sidebar" class="sidebar">
+    {{> components/logo width="u-w-100" logoColor="images/logo-white.png"}}
```
Comment (marceloglacial) [inline src/templates/components/sidebar.hbs]: Move logo to page header

File: src/templates/components/sidebar.hbs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    <ul class="sidebar__list">
+        <li class="sidebar__item">
+            <a href="" class="sidebar__info-link">
```
Comment (marceloglacial) [inline src/templates/components/sidebar.hbs]: Move contact items to footer

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: src/templates/components/footer.hbs (hunk 1 above)
Comment (dutraneto) [reply src/templates/components/footer.hbs:2]: Ill do it! But, if o logo goes in the hero and the address goes in the footer, what are we going to put in the sidebar?

File: src/templates/components/home.hbs (hunk 2 above)
Comment (dutraneto) [reply src/templates/components/home.hbs]: Done

File: src/templates/components/footer.hbs (hunk 1 above)
Comment (dutraneto) [reply src/templates/components/footer.hbs:2]: Done

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**